src/stores/notification.ts

```diff
@@ -1,10 +1,5 @@
-import {
```

Comment:
suggestion: on line with `state()`: I think the return type is no longer required, with the update TS documentation. (see also `breadcrumb`)